start-from-zero/Javascript/src/views/new-evaluation/new-evaluation.js

```diff
@@ -0,0 +1,151 @@
+function NewEvaluationPage(options) {
+    const newOptions = options || {};
+    return `
+    ${NAV()}
+    <section class="form-candidate">
+    <div class="container">
+       <div class="row">
+            <div class="col-md-12">
+                <form action="" class="user-input" id='form-new' method='post'>
+    ${Inputs(
+        InputData()
+    )}
+    ${TechnicalLevelPicker(
+        TechnicalData()
+    )}
+    ${TextArea(
+        TextareaData()
+    )}
+    ${SelectOption(
+        FieldsetData()
+    )}
+                        <div class="button-wrap">
+                            <input type="button" id='btn' class='submit-button' value='Submit'>
+                        </div>
+                    </form>
+                </div>
+            </div>
+        </div>
+    </section>
+    ${Footer()}
+    `;
+}
+
+
+const TableData = function (options = {}) {
+    const currentLevel = options.headers.map((el) => `
+            <td class='technical-level-td'>${el}</td>
+        `).join('');
+    return currentLevel;
+};
+
+const TableCheckbox = function (options = {}) {
+    const checkboxArr = [];
+    options.columnData.forEach((d) => {
+        checkboxArr.push(`
+            <td>
+                ${d.inputLevels.map((j) => `
+                    <input class='input-checkbox' type="radio" name='radio' value='${j}'>
+                `).join('')}
+            </td>
+        `);
+    });
+    return checkboxArr.join('');
+};
+
+
+const TechnicalLevelPicker = function (options = {}) {
+    return `
+    <h4 class='technical-header'>${options.title}</h4>
+    <table class='technical-level-table'>
+        <tbody class='tbody-new'>
+            <tr class='technical-level'>
+                ${TableData({ headers: options.headers })}
+            </tr>
+            <tr class="technical-level-category">
+                ${TableCheckbox({ columnData: options.columnData })}
+            </tr>
+        </tbody>
+    </table>
+    `;
+};
+
+const TextareaElements = function (options = {}) {
+    const textareaArr = [];
+    options.textarea.forEach((d) => {
+        textareaArr.push(`
+            <div class="textarea-wrap">
+            <h4 class='textarea-header'>${d.label}</h4>
+            <textarea class='textarea' name="" id="${d.id}" rows="10" placeholder="${d.placeholder}"></textarea>
+            <div>
+        `);
+    });
+    return textareaArr.join('');
+};
+
+const TextArea = function (options = {}) {
```

Comment:
as far as I can see this function is just a wrapper of the previous function as it simply returns the output of the previous one. Do we need to keep it or can it be removed?

Reply:
I have a bunch of functions declared in this way. I think I will stick with this style for now if it's ok with you.

Reply:
sure thing, I was simply wondering :)